_tests/webpack.config.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
         alias: {
             'my-absolute-test-lib': path.join(__dirname, 'lib/a'),
             'same-folder-lib': path.resolve(__dirname, 'lib/b'),
-        }
+        },
+        extensions: ['', '.js']
```

Comment:
This was my best effort to ensure the `extensionsConf` code path is tested. However, it's possible I broke some other relevant testing.